### DISCUSSION TITLE
vs code copilot is encountering temporary issues

### DISCUSSION THREAD

**Discussion by lxk1997** [community/community#28155] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/28109139/182569239-742e4c8a-8e19-48f1-85b5-4e88b9631266.png)
engine request error~

**Comment by davecheney** [comment]
@lxk1997 I cannot see any telemetry from your editor or evidence of your requests hitting the copilot servers.

Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?
